### ISSUE TITLE
Integer overflow with setFrequency's period

### ISSUE THREAD

**Issue by microsoftsam** [lancaster-university/codal-core#1]
Using our playTone(frequency, ms) method, if you type in a value for the interval that is above 3000 ms, the behaviour on the physical device starts being weird. 

Repro in makecode: 
```
loops.forever(() => {
    music.playTone(262, 5000)
    music.playTone(Note.A, 1000)
})
```

The above code doesn't play the initial tone for 5 seconds, but for a time much shorter. 
If you try to play a tone for around 3000, it will play it forever.

It seems this is because we're expecting the user to type in a value in ms, and then codal converts that to microseconds and later nano seconds, causing an integer overflow, with such low values for the time we want to play a tone.

**Comment by finneyj** [comment]
@abchatra Just built a repro of this in C, so definitely something in the Synthesizer. I'll work up a bugfix.

**Comment by finneyj** [comment]
@microsoftsam Thanks for the hint, you were exactly right.

Fixed in codal-circuit-playground Snapshot 1.2.5

**Comment by microsoftsam** [comment]
Awesome, thanks!
